helm/hpcc/values.schema.json

```diff
@@ -146,14 +156,85 @@
         "storageClass": {
           "type": "string"
         },
+        "plane": {
+          "type": "string"
+        },
         "existingClaim": {
           "type": "string"
         },
         "forcePermissions": {
           "type": "boolean"
         }
       },
-      "required": [ "storageSize" ],
+      "oneOf": [
+        {
+          "required": [ "storageSize" ]
+        },
+        {
+          "required": [ "plane" ]
+        }
+      ],
+      "additionalProperties": false
+    },
+    "storagePlanes": {
+      "description": "storage plane definitions",
+      "oneOf": [
+        {
+          "type": "array",
+          "items": { "$ref": "#/definitions/storagePlane" }
+        },
+        {
+          "type": "null"
+        }
+      ]
+    },
+    "storagePlane": {
+      "description": "information about an individual storage plane",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "the name of the storage plane",
+          "type": "string"
+        },
+        "prefix": {
+          "description": "either the path for a local mount, or the url prefix",
+          "type": "string"
+        },
+        "secret": {
+          "description": "optional name of any secret required to access this storage plane",
+          "type": "string"
+        },
+        "pvc": {
+          "description": "optional name of the persistent volume claim for this plane",
+          "type": "string"
+        },
+        "hosts": {
+          "description": "optional name of the host group (for bare metal storage)",
+          "type": "string"
+        },
+        "numDevices": {
+          "description": "optional name of devices in the storage plane",
+          "type": "integer"
+        },
+        "includeDeviceInPath": {
+          "description": "is a directory based on the part number appended to the path.  May not be needed",
+          "type": "boolean"
+        },
+        "speed": {
+          "description": "the speed of reading the data direct from a local mount/connection",
+          "type": "integer"
+        },
+        "remoteSpeed": {
+          "description": "the speed of reading indirectly",
+          "type": "integer"
+        },
```

Comment:
I think agree, as you were saying, that these type of options are best left/re-introduced later, if/when we start to have heuristics to use them.